square.py

```diff
@@ -19,28 +19,41 @@ def draw(self, surface, camera):
 class Hero(Square):
     ''' The Hero is a creature which is managed by the user '''
     
+    INITIAL_STEP_TIME = 0.15
+    FASTEST_STEP_TIME = 0.04
```

Comment:
It will be better to declare them inside __init__(...)

Reply:
Alright.